### DISCUSSION TITLE
Fields not visible in board view

### DISCUSSION THREAD

**Discussion by jdshkolnik** [community/community#4940] [Projects and Issues] [answered]
Am I misunderstanding something?

![image](https://user-images.githubusercontent.com/2047520/128427381-04211cf8-57b3-4899-99b2-e51fff8347a6.png)

**Comment by azenMatt** [comment, marked as answer]
This one should be fixed now - all the custom fields can be toggled via the menu in your screenshot to determine what you would like to see on the cards. 🎉
